templates/zerver/footer.html

```diff
@@ -1,3 +1,31 @@
+{% if corporate_enabled %}
```

Comment:
This is the opposite of the desired condition. (Maybe you flipped this the wrong way for testing?)